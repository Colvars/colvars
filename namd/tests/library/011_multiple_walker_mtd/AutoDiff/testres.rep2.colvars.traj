# step         d                     fa_d                   
         100    9.94700459588437e+00 -1.70182688913198e-01  
         101    9.94458507772995e+00 -1.70182688913198e-01  
         102    9.94237837514508e+00 -1.70182688913198e-01  
         103    9.94046457996010e+00 -1.70182688913198e-01  
         104    9.93917796988520e+00 -1.70182688913198e-01  
         105    9.93831816048514e+00 -1.70182688913198e-01  
         106    9.93730878764442e+00 -1.70182688913198e-01  
         107    9.93672533620511e+00 -1.70182688913198e-01  
         108    9.93648376088839e+00 -1.70182688913198e-01  
         109    9.93652761776498e+00 -1.70182688913198e-01  
         110    9.93706370695588e+00 -1.86126457148131e-01  
         111    9.93776112075882e+00 -1.86126457148131e-01  
         112    9.93854506029798e+00 -1.86126457148131e-01  
         113    9.93918085186220e+00 -1.86126457148131e-01  
         114    9.93937655398418e+00 -1.86126457148131e-01  
         115    9.93893038386686e+00 -1.86126457148131e-01  
         116    9.93789380210237e+00 -1.86126457148131e-01  
         117    9.93662176845159e+00 -1.86126457148131e-01  
         118    9.93489574805879e+00 -1.86126457148131e-01  
         119    9.93264204174611e+00 -1.86126457148131e-01  
         120    9.93021575754143e+00 -2.01684581228258e-01  
         121    9.92745576666947e+00 -2.01684581228258e-01  
         122    9.92406367114035e+00 -2.01684581228258e-01  
         123    9.91986260200830e+00 -2.01684581228258e-01  
         124    9.91533756814292e+00 -2.01684581228258e-01  
         125    9.91093484240737e+00 -2.01684581228258e-01  
         126    9.90665291579182e+00 -2.01684581228258e-01  
         127    9.90211123152112e+00 -2.01684581228258e-01  
         128    9.89781300213203e+00 -2.01684581228258e-01  
         129    9.89406396885698e+00 -2.01684581228258e-01  
         130    9.89014113666249e+00 -2.14600525240779e-01  
         131    9.88640594373758e+00 -2.14600525240779e-01  
         132    9.88226178497735e+00 -2.14600525240779e-01  
         133    9.87820929310142e+00 -2.14600525240779e-01  
         134    9.87355012458204e+00 -2.14600525240779e-01  
         135    9.86875344554473e+00 -2.14600525240779e-01  
         136    9.86400574168410e+00 -2.14600525240779e-01  
         137    9.85896120013053e+00 -2.14600525240779e-01  
         138    9.85422080643946e+00 -2.14600525240779e-01  
         139    9.84970383929048e+00 -2.14600525240779e-01  
         140    9.84571930339719e+00 -2.23905909555689e-01  
         141    9.84162990482619e+00 -2.23905909555689e-01  
         142    9.83714752694151e+00 -2.23905909555689e-01  
         143    9.83237840269532e+00 -2.23905909555689e-01  
         144    9.82753783768742e+00 -2.23905909555689e-01  
         145    9.82322257146133e+00 -2.23905909555689e-01  
         146    9.81960403148490e+00 -2.23905909555689e-01  
         147    9.81638214984508e+00 -2.23905909555689e-01  
         148    9.81411815280101e+00 -2.23905909555689e-01  
         149    9.81186120171889e+00 -2.23905909555689e-01  
         150    9.81023975732696e+00 -2.29929520564960e-01  
         151    9.80895196413704e+00 -2.29929520564960e-01  
         152    9.80803347273143e+00 -2.29929520564960e-01  
         153    9.80725709506516e+00 -2.29929520564960e-01  
         154    9.80685628234177e+00 -2.29929520564960e-01  
         155    9.80643124017792e+00 -2.29929520564960e-01  
         156    9.80651929062019e+00 -2.29929520564960e-01  
         157    9.80658365740485e+00 -2.29929520564960e-01  
         158    9.80608274371768e+00 -2.29929520564960e-01  
         159    9.80533153641205e+00 -2.29929520564960e-01  
         160    9.80408985591389e+00 -2.35357581465163e-01  
         161    9.80243195378187e+00 -2.35357581465163e-01  
         162    9.80032235258663e+00 -2.35357581465163e-01  
         163    9.79828002164672e+00 -2.35357581465163e-01  
         164    9.79526928693096e+00 -2.35357581465163e-01  
         165    9.79205899981332e+00 -2.35357581465163e-01  
         166    9.78836208455697e+00 -2.35357581465163e-01  
         167    9.78427949242546e+00 -2.35357581465163e-01  
         168    9.77985340193428e+00 -2.35357581465163e-01  
         169    9.77597576729126e+00 -2.35357581465163e-01  
         170    9.77231656680436e+00 -2.37624969895514e-01  
         171    9.76872764540608e+00 -2.37624969895514e-01  
         172    9.76445831928769e+00 -2.37624969895514e-01  
         173    9.76039937757358e+00 -2.37624969895514e-01  
         174    9.75645214354585e+00 -2.37624969895514e-01  
         175    9.75287081367028e+00 -2.37624969895514e-01  
         176    9.74968666022476e+00 -2.37624969895514e-01  
         177    9.74684973072586e+00 -2.37624969895514e-01  
         178    9.74348359335075e+00 -2.37624969895514e-01  
         179    9.74030679423950e+00 -2.37624969895514e-01  
         180    9.73711487809642e+00 -2.36313611463974e-01  
         181    9.73433464376223e+00 -2.36313611463974e-01  
         182    9.73173779878573e+00 -2.36313611463974e-01  
         183    9.72947842909980e+00 -2.36313611463974e-01  
         184    9.72710977092031e+00 -2.36313611463974e-01  
         185    9.72467705766697e+00 -2.36313611463974e-01  
         186    9.72244088949095e+00 -2.36313611463974e-01  
         187    9.72029111920384e+00 -2.36313611463974e-01  
         188    9.71845562125918e+00 -2.36313611463974e-01  
         189    9.71680423295022e+00 -2.36313611463974e-01  
         190    9.71521748810981e+00 -2.32792456745213e-01  
         191    9.71413764529071e+00 -2.32792456745213e-01  
         192    9.71307166091424e+00 -2.32792456745213e-01  
         193    9.71195900641790e+00 -2.32792456745213e-01  
         194    9.71186724184931e+00 -2.32792456745213e-01  
         195    9.71169949999982e+00 -2.32792456745213e-01  
         196    9.71157179938086e+00 -2.32792456745213e-01  
         197    9.71177699258930e+00 -2.32792456745213e-01  
         198    9.71199161585537e+00 -2.32792456745213e-01  
         199    9.71215448992823e+00 -2.32792456745213e-01  
         200    9.71175738207289e+00 -2.28926002246273e-01  
