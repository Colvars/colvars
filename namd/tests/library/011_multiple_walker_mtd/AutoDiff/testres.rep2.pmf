# 1
#          0       0.5        50  0

 0.25  0.187767
 0.75  0.187767
 1.25  0.187767
 1.75  0.187767
 2.25  0.187767
 2.75  0.187767
 3.25  0.187767
 3.75  0.187767
 4.25  0.187767
 4.75  0.187767
 5.25  0.187767
 5.75  0.187767
 6.25  0.187767
 6.75  0.187767
 7.25  0.187767
 7.75  0.187767
 8.25  0.187767
 8.75  0.187294
 9.25  0.15902
 9.75  0.0130218
 10.25  0.0833529
 10.75  0.182076
 11.25  0.187741
 11.75  0.187767
 12.25  0.187767
 12.75  0.187755
 13.25  0.184278
 13.75  0.102893
 14.25  -0
 14.75  0.150318
 15.25  0.187113
 15.75  0.187767
 16.25  0.187767
 16.75  0.187767
 17.25  0.187767
 17.75  0.187767
 18.25  0.187767
 18.75  0.187767
 19.25  0.187767
 19.75  0.187767
 20.25  0.187767
 20.75  0.187767
 21.25  0.187767
 21.75  0.187767
 22.25  0.187767
 22.75  0.187767
 23.25  0.187767
 23.75  0.187767
 24.25  0.187767
 24.75  0.187767
