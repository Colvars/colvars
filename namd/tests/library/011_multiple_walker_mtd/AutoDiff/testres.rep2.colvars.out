colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.rep2.in":
colvars: # units = "" [default]
colvars: # indexFile = "../Common/da.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "d"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "Protein_C-alpha_1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "Protein_C-alpha_10"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = 0
colvars:   # upperBoundary = 25
colvars:   Reading legacy options upperWall and upperWallConstant: consider using a harmonicWalls restraint (caution: force constant would then be scaled by width^2).
colvars:   # upperWall = 25
colvars:   Generating a new harmonicWalls bias for compatibility purposes.
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "metadynamics" instance.
colvars:   # name = "metadynamics1" [default]
colvars:   # colvars = { d }
colvars:   # zeroStepData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 40000 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # hillWeight = 0.01
colvars:   # newHillFrequency = 10
colvars:   # gaussianSigmas =  [default]
colvars:   # hillWidth = 1.25331
colvars:   Half-widths of the Gaussian hills (sigma's):
colvars:   d: 0.313329
colvars:   # multipleReplicas = on
colvars:   # useGrids = on [default]
colvars:   # gridsUpdateFrequency = 10 [default]
colvars:   # rebinGrids = off [default]
colvars:   # writeFreeEnergyFile = on [default]
colvars:   # keepHills = off [default]
colvars:   # keepFreeEnergyFiles = off [default]
colvars:   # writeHillsTrajectory = off [default]
colvars:   # writePartialFreeEnergyFile = off [default]
colvars:   # replicaID = "rep2"
colvars:   # replicasRegistry = "replicas.registry.txt"
colvars:   # replicaUpdateFrequency = 10
colvars:   # wellTempered = off [default]
colvars:   # biasTemperature = -1 [default]
colvars:   # ebMeta = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1 [default]
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonicwalls" instance.
colvars:   # name = "dw"
colvars:   # colvars = { d }
colvars:   # zeroStepData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 40000 [default]
colvars:   # timeStepFactor = 1
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # forceConstant = 1 [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   # lowerWalls = { 0 } [default]
colvars:   Lower walls were not provided.
colvars:   # upperWalls = { 25 }
colvars:   # upperWallConstant = 25
colvars:   The upper wall force constant for colvar "d" will be rescaled to 100 according to the specified width (0.5).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: ----------------------------------------------------------------------
colvars: Restarting from file "test.rep2.colvars.state".
colvars:   Restarting collective variable "d" from value: 9.947
colvars:   Restarting metadynamics bias "metadynamics1" from step number 100.
colvars:   Restarting harmonicwalls bias "dw" from step number 100.
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "testres.rep2r.colvars.state".
colvars: The final output state file will be "testres.rep2.colvars.state".
colvars: Opening trajectory file "testres.rep2.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: The restart output state file will be "testres.rep2r.colvars.state".
colvars: The final output state file will be "testres.rep2.colvars.state".
colvars:   Metadynamics bias "metadynamics1": accessing replica "rep1".
colvars:   Metadynamics bias "metadynamics1": replica "rep1" has supplied a new state file, "/data/local/Fiorin/work/colvars/namd/tests/library/011_multiple_walker_mtd/testres.rep1.colvars.metadynamics1.rep1.state".
colvars:   Metadynamics bias "metadynamics1": reading the state of replica "rep1" from file "/data/local/Fiorin/work/colvars/namd/tests/library/011_multiple_walker_mtd/testres.rep1.colvars.metadynamics1.rep1.state".
colvars:   Restarting metadynamics bias "metadynamics1" from step number 200.
colvars: Saving collective variables state to "testres.rep2.colvars.state".
